Remotion/Web/UnitTests/Core/Resources/TestableResourcePathBuilder.cs

```diff
@@ -24,8 +24,8 @@ public class TestableResourcePathBuilder : ResourcePathBuilder
   {
     private IHttpContextProvider _httpContextProvider;
 
-    public TestableResourcePathBuilder (IHttpContextProvider httpContextProvider, string configuredResourceRoot)
-        : base(httpContextProvider, configuredResourceRoot)
+    public TestableResourcePathBuilder (IHttpContextProvider httpContextProvider, ResourceRoot resourceRoot)
```

Comment:
This type is now redudnant. we can put the public property for HttpContextProvider onto the ResourcePathBuilder (and do the same with ResourceRoot. Then drop this type. 

Originally, it was needed because we needed to have a public ctor that accepts the resourceroot. that's now solved in the new API.